odbc/src/api/definitions.rs

```diff
@@ -322,6 +502,64 @@ pub enum DiagType {
     SQL_DIAG_COLUMN_NUMBER = -1247,
 }
 
+#[allow(non_camel_case_types)]
+#[repr(i16)]
+#[derive(Debug, PartialEq, Eq, Clone, Copy, FromPrimitive)]
+pub enum ParamType {
```

Comment:
The following few enums are newly added but technically unused for now. These are all used by functions we have not yet (or may never) implement. I can remove these from this PR and we can address those functions if/when we implement them. Let me know what you prefer.